Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nom-exif"
-version = "1.2.6"
+version = "1.3.0"
```

Comment:
I chose to bump minor since this is kind of a new feature, but I don't feel super-strongly about that.